app/src/androidTestKiwix/java/org/kiwix/kiwixmobile/help/HelpActivityTest.java

```diff
@@ -85,7 +85,7 @@ public void testHelpActivity() {
     onView(withText(test)).check(matches(notNullValue()));
 
     onView(withText(context.getString(R.string.help_12))).check(matches(notNullValue()));
-    onView(withText(context.getString(R.string.help_12))).perform(click());
+  //  onView(withText(context.getString(R.string.help_12))).perform(click());
```

Comment:
Uncomment this please

Reply:
I did not mean delete, there should be no changes to this test file because you did not alter HelpActivity.
Sorry I should have been more clear and said revert your changes